### PR TITLE
add blurb about jsQR

### PR DESCRIPTION
**Pull request by NullVoxPopuli**
does the GH PR template need to be updated for ember-times submissions? idk how'd that work, but nothing made sense, so I deleted it all :-/

### CODE REVIEW

**Review by MinThaMie — CHANGES_REQUESTED** [review]
You forgot to add your part to the intro :(

**Comment by NullVoxPopuli** [comment]
What intro? :/

File: source/2020-03-13-the-ember-times-issue-139.md [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
 title: The Ember Times - Issue No. 139
-author: Abhilash LR, the crowd
+author: Abhilash LR, Preston S, the crowd
 tags: Recent Posts, Newsletter, Ember.js Times, Ember Times, 2020
 alias : "blog/2020/03/13-the-ember-times-issue-139.html"
 responsive: true
 ---
 
 வணக்கம் Emberistas! 🐹
+QR scanning without bundle size growing!
```
Comment (MinThaMie) [inline source/2020-03-13-the-ember-times-issue-139.md]:
```suggestion
QR scanning without bundle size growing 📷!
```

**Review by MinThaMie — APPROVED** [review]

**Comment by ijlee2** [comment]
I'll merge this so that I and others can branch off from the latest, thanks.

**Pull request merged**